src/main/resources/templates/fragments/topnav.html

```diff
@@ -1,17 +1,21 @@
 <div th:fragment="navbar" xmlns:th="http://www.w3.org/1999/xhtml">
-  <nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <h2> Meal Mentor </h2>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
+    <style>
+        h2 {
+            color: #2faf9f;
+        }
+
+        li {
+            padding-left: 70px;
+        }
+    </style>
 
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav mr-auto">
         <li class="nav-item active">
-          <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+          <a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
         </li>
         <li class="nav-item active">
-          <a class="nav-link" href="#">Shopping List</a>
+          <a class="nav-link" href="/">Shopping List</a>
```

Comment:
The # reference does not work to redirect to the home page which added to the being stuck on the error page. This adds the route back to the home page.